src/Components/BubbleList.svelte

```diff
@@ -2,32 +2,33 @@
 <!-- ability to delete contact-->
 <!-- change contact "label"-->
 
-<script>import BubbleCard from "./Bubble/BubbleCard.svelte";
+<script>
+import {tokenToString} from "../git/token";
+import BubbleCard from "./Bubble/BubbleCard.svelte";
 import BubbleContacts from "./Bubble/BubbleContacts.svelte";
-
+  import { Plus } from 'svelte-hero-icons';
+import { onMount } from "svelte";
+import Relationship from "../git/relationship";
   // get all contacts
   const allBubbles = [
     {
-      name: "Friends",
+      name: "my-relationship",
       contacts: [
-        {
-          name: "Michiel Barrymore",
-          pic: null
-        },
-        {
-          name: "Yves Nova",
-          pic: null
-        },
-        {
-          name: "Tamara Tomoro",
-          pic: null
-        }
       ]
     },
     {
       name: "osoc ladies"
     }
   ];
+let res;
+onMount(async () => {
+  let relationship = await Relationship.get("my-relationship");
+  res = await Promise.all((await relationship.getTokens()).map(tokenToString));
+   res.forEach((elem) => allBubbles[0].contacts.push({"name": elem, "pic": null}));
+   console.info("TAK", allBubbles[0].contacts);
+});
+  // get all contacts
+ 
   
```

Comment:
Maybe just:
```js
onMount(async () => {
  let relationship = await Relationship.get("my-relationship");
  for (let token of await relationship.getTokens()) {
    let tokenString = tokenToString(token);
    allBubbles[0].contacts.push({"name": tokenString, "pic": null});
  }
   console.info("TAK", allBubbles[0].contacts);
});
```
(probably my fault)

Reply:
This should also invalidate the list if it's already expanded. Right now, if you open the page and expand the contacts list of a bubble before the members have been fetched, you have to collapse and expand it again to show the contacts.